src/Controller/WebInterfaceController.php

```
@@ -4,14 +4,84 @@
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\BufferedOutput;
+use Symfony\Component\Console\Command\Command;
+use App\Command\TestSearchCommand;
+use App\Command\ProcessImageCommand;
 use Symfony\Component\Routing\Annotation\Route;
 
 class WebInterfaceController extends AbstractController
 {
+    private const MAX_QUERY_LENGTH = 500;
+    private const MAX_IMAGE_SIZE = 5242880; // 5 MB
```

PHPStan (2.1.15)

Constant App\Controller\WebInterfaceController::MAX_IMAGE_SIZE is unused.
See: https://phpstan.org/developing-extensions/always-used-class-constants

(classConstant.unused)